src/styles/blocks/articles.css

```
@@ -12,6 +12,110 @@
     font-size: 21px;
 }
 
+@media screen and (max-width: 320px) {
+
+    .main{
+        box-sizing: border-box;
+        display: grid;
+    }
+
+
+    h1{
+        position: relative;
+
+        font-family: RF Dewi;
+        font-size: 21px;
+        line-height: 25px;
+        letter-spacing: 0.03em;
+        color: #FFFFFF;
+
+    }
+
+    h2{
+        font-family: RF Dewi Expanded;
+        font-size: 16px;
+        line-height: 19px;
+        letter-spacing: 0.08em;
+        text-transform: uppercase;
+
+        color: #000000;
+        display: flex;
+        justify-content: flex-start;
+    }
+
+
+    .lead:first-letter{
+        font-family: RF Dewi Expanded;
+        font-size: 73px;
+        /*line-height: 124px;*/
+
+        letter-spacing: 0.033em;
+        color: #696969;
+
+
+        padding: 3px; /* Поля вокруг первой буквы */
+        margin-right: 5px; /* Отступ справа */
+        float: left; /* Обтекание по правому краю */
+
+
+    }
+
+    .lead {
+        clear: left;
+    }
+
+    p{
+        font-family: RF Dewi;
+        font-size: 12px;
+        line-height: 170%;
+        /* or 20px */
+
+
+        color: #000000;
+    }
+    ol{
+        list-style-type: #B1B3B6;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-around;
+
+    }
+
+}
+
+@media screen and (min-width: 320px, max-width: 1024px) {
```

Может быть вы это имели в виду? Буквально: стили применятся только от 320 до 1024. В любом случае, запятую и несколько условий внутри скобок не ставят.

```suggestion
@media (min-width: 320px) and (max-width: 1024px) {
```